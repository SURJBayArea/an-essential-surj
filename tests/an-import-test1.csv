nationbuilder_id,first_name,last_name,email,email_opt_in,email1,email1_is_bad,email2,email2_is_bad,email3,email3_is_bad,email4,email4_is_bad,unsubscribed_at,phone_number,work_phone_number,mobile_number,mobile_opt_in,is_mobile_bad,do_not_call,do_not_contact,federal_donotcall,facebook_username,twitter_login,meetup_id,primary_address1,primary_address2,primary_address3,primary_city,primary_county,primary_state,primary_zip,primary_country_code,primary_country,primary_fips,primary_submitted_address,address_address1,address_address2,address_address3,address_city,address_county,address_state,address_zip,address_country_code,address_country,address_fips,address_submitted_address,registered_address1,registered_address2,registered_address3,registered_city,registered_county,registered_state,registered_zip,registered_country_code,registered_country,registered_street_number,registered_street_prefix,registered_street_name,registered_street_type,registered_street_suffix,registered_unit_number,registered_zip5,registered_zip4,registered_fips,registered_sort_sequence,registered_delivery_point,registered_lot,registered_carrier_route,registered_submitted_address,billing_address1,billing_address2,billing_address3,billing_city,billing_county,billing_state,billing_zip,billing_country_code,billing_country,billing_fips,billing_submitted_address,work_address1,work_address2,work_address3,work_city,work_county,work_state,work_zip,work_country_code,work_country,work_fips,work_submitted_address,mailing_address1,mailing_address2,mailing_address3,mailing_city,mailing_county,mailing_state,mailing_zip,mailing_country_code,mailing_country,mailing_street_number,mailing_street_prefix,mailing_street_name,mailing_street_type,mailing_street_suffix,mailing_unit_number,mailing_zip5,mailing_zip4,mailing_fips,mailing_sort_sequence,mailing_delivery_point,mailing_lot,mailing_carrier_route,mailing_submitted_address,user_submitted_address1,user_submitted_address2,user_submitted_address3,user_submitted_city,user_submitted_county,user_submitted_state,user_submitted_zip,user_submitted_country_code,user_submitted_country,user_submitted_fips,user_submitted_submitted_address,supranational_district,federal_district,state_upper_district,state_lower_district,county_district,city_district,city_sub_district,village_district,judicial_district,school_district,school_sub_district,fire_district,precinct_name,precinct_code,media_market_name,signup_type,tag_list,note,employer,occupation,marital_status,sex,demo,ethnicity,language,religion,church,born_at,is_donor,is_fundraiser,is_ignore_donation_limits,first_donated_at,last_donated_at,donations_count,donations_amount,donations_raised_count,donations_raised_amount,donations_pledged_amount,donations_count_this_cycle,donations_amount_this_cycle,donations_raised_count_this_cycle,donations_raised_amount_this_cycle,party,previous_party,inferred_party,party_member,registered_at,is_active_voter,is_early_voter,is_absentee_voter,is_permanent_absentee_voter,support_probability_score,turnout_probability_score,is_customer,closed_invoices_amount,outstanding_invoices_amount,is_volunteer,availability,membership_names,faith_leader,clergy_type,chapter,#Call Representatives,?Web Design,!Phone Bank,tag:Request Contact
97439,Deven,Mc No Email,,TRUE,demo+supporter1@surjaction.org,FALSE,,FALSE,,FALSE,,FALSE,,510 555-5555,,510 555-5555,TRUE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,"!Phone Bank, #Militarization, Demo - Interest - Human Billboard, Demo - Origin - Action 2017-03, Demo - Origin - Website, @Demo",,,,,,,,,,,,FALSE,FALSE,FALSE,,,0,$0.00 ,0,$0.00 ,$0.00 ,0,$0.00 ,0,$0.00 ,,,,FALSE,,,,,,,,FALSE,,,FALSE,,,,,,Yes,,,
97444,Demo,,demo+supporter2@surjaction.org,TRUE,demo+supporter2@surjaction.org,FALSE,,FALSE,,FALSE,,FALSE,,,,,TRUE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,US,United States,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,US,United States,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,"#Militarization, Demo - Origin - Website, @Demo",,,,,,,,,,,,TRUE,FALSE,FALSE,8/11/16 0:46,8/11/16 0:46,1,$10.00 ,0,$0.00 ,$0.00 ,0,$0.00 ,0,$0.00 ,,,,FALSE,,,,,,,,FALSE,,,FALSE,,,,,,,,,x
97475,Demo,Basebuilder1,demo+basebuilder1@surjaction.org,TRUE,demo+basebuilder1@surjaction.org,FALSE,,FALSE,,FALSE,,FALSE,,510 555-5555,,510 555-5555,FALSE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,"Demo - Origin - Action 2017-03, Demo - Origin - Website, @Demo",,,,,,,,,,,,FALSE,FALSE,FALSE,,,0,$0.00 ,0,$0.00 ,$0.00 ,0,$0.00 ,0,$0.00 ,,,,FALSE,,,,,,,,FALSE,,,FALSE,,,,,,Yes,X,,
97500,Demo,Fundraiser2,demo+fundraiser2@surjaction.org,TRUE,demo+fundraiser2@surjaction.org,FALSE,,FALSE,,FALSE,,FALSE,,,,,TRUE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,"?Fundraising, Demo - Origin - Website, @Demo",,Royal Jay Ltd,,,,,,,,,,FALSE,TRUE,FALSE,,,0,$0.00 ,0,$0.00 ,$0.00 ,0,$0.00 ,0,$0.00 ,,,,FALSE,,,,,,,,FALSE,,,FALSE,,,,,,No,X,,
97503,Demo,Fundraiser3,demo+fundraiser3@surjaction.org,TRUE,demo+fundraiser3@surjaction.org,FALSE,,FALSE,,FALSE,,FALSE,,,,,TRUE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,"?Fundraising, !Phone Bank, Demo - Origin - Action 2017-03, Demo - Origin - Website, @Demo",,,,,,,,,,,,FALSE,TRUE,FALSE,,,0,$0.00 ,0,$0.00 ,$0.00 ,0,$0.00 ,0,$0.00 ,,,,FALSE,,,,,,,,FALSE,,,FALSE,,,,,,FALSE,X,,
97540,Demo,Basebuilder2,demo+basebuilder2@surjaction.org,TRUE,demo+basebuilder2@surjaction.org,FALSE,,FALSE,,FALSE,,FALSE,,,,,TRUE,FALSE,FALSE,FALSE,FALSE,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,"Demo - Interest - Human Billboard, !Direct Action, Demo - Origin - Website, @Demo",,,,,,,,,,,,FALSE,FALSE,FALSE,,,0,$0.00 ,0,$0.00 ,$0.00 ,0,$0.00 ,0,$0.00 ,,,,FALSE,,,,,,,,FALSE,,,FALSE,,,,,,,,,
97541,Demo,Mc Emali Opt Out,demo+basebuilder3@surjaction.org,FALSE,demo+basebuilder3@surjaction.org,FALSE,,FALSE,,FALSE,,FALSE,,,,,TRUE,FALSE,FALSE,FALSE,FALSE,,,,,,,Emeryville,,CA,94608,US,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,"Demo - Interest - Human Billboard, Demo - Origin - Website, @Demo",,,,,,,,,,,,FALSE,FALSE,FALSE,,,0,$0.00 ,0,$0.00 ,$0.00 ,0,$0.00 ,0,$0.00 ,,,,FALSE,,,,,,,,FALSE,,,FALSE,,,,,,,,,x
97552,Demo,Supporter3,demo+supporter3@surjaction.org,TRUE,demo+supporter3@surjaction.org,FALSE,,FALSE,,FALSE,,FALSE,,510 123-4567,,510 123-4567,TRUE,FALSE,FALSE,FALSE,FALSE,,,,4000 Railway Rd,Suite 399,,Oakland,Alameda,CA,94608,US,United States,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,"!Direct Action, Demo - Origin - Website, @Demo",,Racial Justice LLC,,,,,,,,,,FALSE,FALSE,FALSE,,,0,$0.00 ,0,$0.00 ,$0.00 ,0,$0.00 ,0,$0.00 ,,,,,,,,,,,,FALSE,,,FALSE,,,,,,,,,x