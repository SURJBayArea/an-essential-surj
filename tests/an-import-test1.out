[PREP] an_profiles: SURJ Demo Chapter: API Token found
[PREP] FOUND COLUMN TAGS: {} {'#Call Representatives': '#Call Representatives', '?Web Design': '?Web Design', '!Phone Bank': '!Phone Bank', 'tag:Request Contact': 'Request Contact'}
[001] TRY Mc No Email, Deven <> OPT-IN TRUE WITH TAGS !Phone Bank, #Militarization, Demo - Interest - Human Billboard, Demo - Origin - Action 2017-03, Demo - Origin - Website, @Demo
[001] SKIP - NO EMAIL
[002] TRY , Demo <demo+supporter2@surjaction.org> OPT-IN TRUE WITH TAGS #Militarization, Demo - Origin - Website, @Demo
[002] OK {
    "person": {
        "email_addresses": [
            {
                "address": "demo+supporter2@surjaction.org"
            }
        ],
        "given_name": "Demo"
    },
    "add_tags": [
        "#Militarization",
        "Request Contact"
    ]
}
[003] TRY Basebuilder1, Demo <demo+basebuilder1@surjaction.org> OPT-IN TRUE WITH TAGS Demo - Origin - Action 2017-03, Demo - Origin - Website, @Demo
[003] OK {
    "person": {
        "email_addresses": [
            {
                "address": "demo+basebuilder1@surjaction.org"
            }
        ],
        "family_name": "Basebuilder1",
        "given_name": "Demo"
    },
    "add_tags": [
        "#Call Representatives",
        "?Web Design"
    ],
    "custom_fields": {
        "Phone": "510 555-5555"
    }
}
[004] TRY Fundraiser2, Demo <demo+fundraiser2@surjaction.org> OPT-IN TRUE WITH TAGS ?Fundraising, Demo - Origin - Website, @Demo
[004] OK {
    "person": {
        "email_addresses": [
            {
                "address": "demo+fundraiser2@surjaction.org"
            }
        ],
        "family_name": "Fundraiser2",
        "given_name": "Demo"
    },
    "add_tags": [
        "?Fundraising",
        "?Web Design"
    ],
    "custom_fields": {
        "organization": "Royal Jay Ltd"
    }
}
[005] TRY Fundraiser3, Demo <demo+fundraiser3@surjaction.org> OPT-IN TRUE WITH TAGS ?Fundraising, !Phone Bank, Demo - Origin - Action 2017-03, Demo - Origin - Website, @Demo
[005] OK {
    "person": {
        "email_addresses": [
            {
                "address": "demo+fundraiser3@surjaction.org"
            }
        ],
        "family_name": "Fundraiser3",
        "given_name": "Demo"
    },
    "add_tags": [
        "?Fundraising",
        "!Phone Bank",
        "?Web Design"
    ]
}
[006] TRY Basebuilder2, Demo <demo+basebuilder2@surjaction.org> OPT-IN TRUE WITH TAGS Demo - Interest - Human Billboard, !Direct Action, Demo - Origin - Website, @Demo
[006] OK {
    "person": {
        "email_addresses": [
            {
                "address": "demo+basebuilder2@surjaction.org"
            }
        ],
        "family_name": "Basebuilder2",
        "given_name": "Demo"
    },
    "add_tags": [
        "!Direct Action"
    ]
}
[007] TRY Mc Emali Opt Out, Demo <demo+basebuilder3@surjaction.org> OPT-IN FALSE WITH TAGS Demo - Interest - Human Billboard, Demo - Origin - Website, @Demo
[007] SKIP - UNSUBSCRIBED
[008] TRY Supporter3, Demo <demo+supporter3@surjaction.org> OPT-IN TRUE WITH TAGS !Direct Action, Demo - Origin - Website, @Demo
[008] OK {
    "person": {
        "email_addresses": [
            {
                "address": "demo+supporter3@surjaction.org"
            }
        ],
        "family_name": "Supporter3",
        "given_name": "Demo",
        "postal_addresses": [
            {
                "address_lines": [
                    "4000 Railway Rd",
                    "Suite 399"
                ],
                "locality": "Oakland",
                "region": "CA",
                "postal_code": "94608",
                "primary": true
            }
        ]
    },
    "add_tags": [
        "!Direct Action",
        "Request Contact"
    ],
    "custom_fields": {
        "mobile": "510 123-4567",
        "Phone": "510 123-4567",
        "organization": "Racial Justice LLC"
    }
}
Processed 6 activists in 0:00:00.001426 4208/sec.
